examples/return-opt/index.rsh

```diff
@@ -0,0 +1,28 @@
+'reach 0.1';
```

Comment:
When you run `make hs-test`, this will error because there's no `index.txt` in the repo